src/backend/src/utils/transporter.utils.ts

```diff
@@ -63,4 +66,40 @@ const sendMailToAdvisor = async (subject: string, text: string) => {
   }
 };
 
-export default sendMailToAdvisor;
+export const uploadFile = async (fileObject: Express.Multer.File) => {
```

Comment:
I think this should get its own file or we rename the file

Reply:
also I think you should put a link to whatever tutorial you used to get this to work here.